packages/create-app/README.md

```diff
@@ -8,13 +8,19 @@
 With NPM:
 
 ```bash
-$ npm init @vitejs/app
```

Comment:
Could you re-add the `$ `?
I think they are a stylistic choice to indicate that these are terminal **inputs**
Also they have nothing to do with our PR itself